projects/poppler/build_fuzzer.sh

```diff
@@ -0,0 +1,306 @@
+    CPPFLAGS=-I$PREFIX/include LDFLAGS=-L$PREFIX/lib ./configure --prefix="$PREFIX" --disable-shared --disable-dependency-tracking
+    make -j$(nproc)
+    make install
+
+    pushd $SRC/cairo
+    meson \
+        --prefix=$PREFIX \
+        --libdir=lib \
+        --default-library=static \
+        _builddir
+    ninja -C _builddir
+    ninja -C _builddir install
+    popd
+
+    pushd $SRC/pango
+    meson \
+        -Ddefault_library=static \
+        --prefix=$PREFIX \
+        --libdir=lib \
+        _builddir
+    sed -i -e 's/ -Werror=implicit-fallthrough//g' _builddir/build.ninja
+    ninja -C _builddir
+    ninja -C _builddir install
+    popd
+fi
+
+pushd $SRC/qtbase
+# add the flags to Qt build too
+sed -i -e "s/QMAKE_CXXFLAGS    += -stdlib=libc++/QMAKE_CXXFLAGS    +=  $CXXFLAGS\nQMAKE_CFLAGS += $CFLAGS/g" mkspecs/linux-clang-libc++/qmake.conf
+sed -i -e "s/QMAKE_LFLAGS      += -stdlib=libc++/QMAKE_LFLAGS      +=  -lpthread $CXXFLAGS/g" mkspecs/linux-clang-libc++/qmake.conf
+# disable sanitize=vptr for harfbuzz since it compiles without rtti
+sed -i -e "s/TARGET = qtharfbuzz/TARGET = qtharfbuzz\nQMAKE_CXXFLAGS += -fno-sanitize=vptr/g" src/3rdparty/harfbuzz-ng/harfbuzz-ng.pro
+# make qmake compile faster
+sed -i -e "s/MAKE\")/MAKE\" -j$(nproc))/g" configure
+# Fix memory stuff in qt 5.15 unfixable since branch is closed now
+sed -i -e "s/struct statx statxBuffer/struct statx statxBuffer = {}/g" src/corelib/io/qfilesystemengine_unix.cpp
+./configure --glib=no --libpng=qt -opensource -confirm-license -static -no-opengl -no-icu -no-pkg-config -platform linux-clang-libc++ -nomake tests -nomake examples -prefix $PREFIX -D QT_NO_DEPRECATED_WARNINGS
+make -j$(nproc)
+make install
+popd
+
+# Poppler complains when PKG_CONFIG is set to `which pkg-config --static` so
+# temporarily removing it
+export PKG_CONFIG="`which pkg-config`"
+
+if [ "$SANITIZER" != "memory" ]; then
+    POPPLER_ENABLE_GLIB=ON
+    POPPLER_FONT_CONFIGURATION=fontconfig
+else
+    POPPLER_ENABLE_GLIB=OFF
+    POPPLER_FONT_CONFIGURATION=generic
+fi
+
+mkdir -p $SRC/poppler/build
+pushd $SRC/poppler/build
+cmake .. \
+  -DCMAKE_BUILD_TYPE=debug \
+  -DBUILD_SHARED_LIBS=OFF \
+  -DENABLE_FUZZER=OFF \
+  -DFONT_CONFIGURATION=$POPPLER_FONT_CONFIGURATION \
+  -DENABLE_DCTDECODER=none \
+  -DENABLE_GOBJECT_INTROSPECTION=OFF \
+  -DENABLE_LIBPNG=OFF \
+  -DENABLE_ZLIB=OFF \
+  -DENABLE_LIBTIFF=OFF \
+  -DENABLE_LIBJPEG=OFF \
+  -DENABLE_GLIB=$POPPLER_ENABLE_GLIB \
+  -DENABLE_LIBCURL=OFF \
+  -DENABLE_QT5=ON \
+  -DENABLE_UTILS=OFF \
+  -DWITH_Cairo=$POPPLER_ENABLE_GLIB \
+  -DCMAKE_INSTALL_PREFIX=$PREFIX
+
+export PKG_CONFIG="`which pkg-config` --static"
+make -j$(nproc) poppler poppler-cpp poppler-qt5
+if [ "$SANITIZER" != "memory" ]; then
+    make -j$(nproc) poppler-glib
+fi
+
+export CFLAGS="-g -fsanitize=fuzzer,address,integer,bounds,null,undefined,float-divide-by-zero"
+export CXXFLAGS="-g -fsanitize=fuzzer,address,integer,bounds,null,undefined,float-divide-by-zero"
```

Comment:
CFLAGS and CXXFLAGS are already defined

Reply:
same for other sh scripts

Reply:
I use "-fsanitize=fuzzer" here instead "-fsanitize=fuzzer-no-link" as it was in the begging